manifest.json

```
@@ -16,15 +16,15 @@
     },
```

Can't suggest in the correct spot 😉 

```suggestion
      "address": "14egjs9cSxt4jnaZTYQE4NemQVXL6qtayqNNpy3VJPSS6AAcaUq",
      "signature": "fB0pnm5QqRPD8rDiOykRSV6UdICf30SWPb0K3G4VL01skIu34YLWnoZM9voWI/hBQXOfg/bhbhN1028DwbLYAw=="
```

---

@6CornerDefender can you integrate this signature?